tsinfer/formats.py

```diff
@@ -1971,14 +2020,22 @@ def sites(self, ids=None):
         alleles = self.sites_alleles[:]
         metadata = self.sites_metadata[:]
         time = self.sites_time[:]
+        ancestral_allele_index = self.sites_ancestral_allele_index[:]
         if ids is None:
             ids = np.arange(0, self.num_sites, dtype=int)
         for j in ids:
+            aa = ancestral_allele_index[j]
+            al_list = alleles[j]
+            if aa > 0:
+                # reorder the alleles to put the ancestral one first
+                al_list = tuple([al_list[aa]] + al_list[:aa] + al_list[aa + 1 :])
             site = Site(
                 id=j,
                 position=position[j],
-                ancestral_state=alleles[j][0],
-                alleles=tuple(alleles[j]),
+                ancestral_state=(
```

Comment:
I don't think giving the ancestral state back here helps much. Let's just return the straight ``ancestral_allele_index`` that we store.

Reply:
OK. This was to match the tskit `Site` object, though.